src/domain/entities/User.ts

```diff
@@ -10,11 +10,12 @@ export interface User {
     email: string;
     lastUpdated: Date;
     created: Date;
+    apiUrl: string;
     userRoles: UserRole[];
     userGroups: NamedRef[];
     organisationUnits: NamedRef[];
     dataViewOrganisationUnits: NamedRef[];
-    lastLogin: Date;
+    lastLogin: Date | string;
```

Comment:
In domain ``lastLogin`` should be a ``Date``, when parsing/encoding to JSON (data repository implementation) dates should be properly formatted (either manually or with a schema model)